fees/ether-fi/index.ts

```diff
@@ -17,6 +22,72 @@ const getTotalSupply = async (options, target) => {
   });
 };
 
+const getStethFees = async (options, totalSteth) => {
+  const stethRebaseLogs = await options.getLogs({
+    target: STETH,
+    fromBlock: await options.getStartBlock(),
+    toBlock: await options.getEndBlock(),
```

Comment:
from & toBlock can be skipped, it is automatically pulled

Reply:
*can you remove line 28 & 29?

Reply:
done